libs/community/langchain_community/chat_models/solar.py

```diff
@@ -0,0 +1,53 @@
+"""Wrapper around Solar chat models."""
+from typing import Dict
+
+from langchain_core.pydantic_v1 import root_validator
+from langchain_core.utils import get_from_dict_or_env
+
+from langchain_community.chat_models import ChatOpenAI
+from langchain_community.llms.solar import SOLAR_SERVICE_URL_BASE, SolarCommon
+
+
+class SolarChat(SolarCommon, ChatOpenAI):
+    """Wrapper around Solar large language models.
+    To use, you should have the ``openai`` python package installed, and the
+    environment variable ``SOLAR_API_KEY`` set with your API key.
+    (Solar's chat API is compatible with OpenAI's SDK.)
+    Referenced from https://platform.solar.cn/docs
```

Comment:
The reference URL is not working. Please fix it.

Reply:
ok, will change to upstage docs.